abb.repos

```diff
@@ -17,5 +17,5 @@ repositories:
     version: rolling
   thirdparty/abb/abb_ros2:
     type: git
-    url: https://github.com/yadunund/abb_ros2.git
-    version: main
+    url: https://github.com/luca-della-vedova/abb_ros2.git
```

Comment:
This can be reverted now that the PR from your fork was merged.

Reply:
https://github.com/osrf/nexus/pull/35/commits/3d1a51dc8b3d789a296d56741e76df60ac6e6abb